recipes/agg/all/conanfile.py

```diff
@@ -0,0 +1,140 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.errors import ConanInvalidConfiguration
+
+import os
+
+required_conan_version = ">=1.52.0"
```

Comment:
for rm_safe suggestion

```suggestion
required_conan_version = ">=1.53.0"
```